canonical/ubuntu.com
# chore: add desktop flavours images on /desktop/flavours

## Done

- replaced ubuntu desktop flavours images on `/desktop/flavours`

## QA

- Go to https://ubuntu-com-14400.demos.haus/desktop/flavours
- Check if the images from [copy doc](https://docs.google.com/document/d/1M5nkigcSRzq1YVjiXOZkeU4EUCqT0uYBR3Y_sjxM-wM/edit) are updated correctly
- **note**: `Edubuntu` image update is done in another PR.

## Issue / Card

Fixes #https://warthogs.atlassian.net/browse/WD-14870
